README.md

```diff
@@ -110,6 +153,27 @@ Usage
 3. Input the [Device Farm Run Configuration](https://docs.aws.amazon.com/devicefarm/latest/developerguide/test-runs.html#test-runs-configuration)
 4. Click "Generate Pipeline Script"
 
+## TroubleShooting
+
+# "Invalid Credentials" Error while validating Credentials
+
+We validate two things: 1) If the credentials are valid, 2) and if they have access to AWS Device Farm
+
+1. Log in to your jenkins host.
+2. If using access and secret key,
+Run "aws devicefarm list-projects" on the host running jenkins.
```

Comment:
Run "aws devicefarm list-projects" on the host running Jenkins using the AWS CLI

Reply:
Changed